samples/04_gis_analysts_data_scientists/drive_time_analysis_for_opioid_epidemic.ipynb

```diff
@@ -11,14 +11,55 @@
    "cell_type": "markdown",
```

Comment:
<p>missing the word 'centers' after prevention</p>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/421/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>